census/src/test/scala/model/RssdToMsaSpec.scala

```diff
@@ -0,0 +1,13 @@
+package model
+
+import hmda.census.model.RssdToMsa
+import org.scalatest.{ MustMatchers, WordSpec }
+
+class RssdToMsaSpec extends WordSpec with MustMatchers {
+  "RssdToMsa" must {
+    "map values correctly" in {
```

Comment:
I'd also like to see a test that actually confirms that values get mapped directly. Like, choose a few items from the file and make sure they have correct values in the map.